packages/backend/src/index.ts

```diff
@@ -202,6 +203,13 @@ async function main() {
     router: jenkins,
     isOptional: true,
   });
+  await addPlugin({
+    plugin: 'permission',
```

Comment:
```suggestion
    plugin: 'permissions',
```

The name needs to be updated to correlate with `enabled.permissions`.